docs/user/openstack/README.md

```diff
@@ -501,6 +501,12 @@ spec:
           availabilityZone: <optional_openstack_availability_zone>
 ```
 
+[provider-spec-definition]: https://github.com/openshift/cluster-api-provider-openstack/blob/155384b859c5b2fb5b7f11c9111d3f8e4f3066bd/pkg/apis/openstackproviderconfig/v1alpha1/types.go#L31
+
+#### Defining a New MachineSet With Multiple Networks
+
+When attaching multiple networks to a machineset, the primarySubnet in the `providerSpec` need to be set to the `machines subnet` or the node will be unable to join the cluster. The primary subnet is where a nodes kuberentes endpoints are published. Once this is properly set, you can should add all of the networks that you want attached to the machines in the `Networks` list in the `providerSpec`. Note that the network that the primary subnet is part of should also be added to this list.
```

Comment:
```suggestion
To attach multiple networks to a machine set, the `primarySubnet` value in `providerSpec` must be set to the `machines subnet`. If the value is not set, the node cannot join the cluster. 

Nodes' Kubernetes endpoints are published on the primary subnet. After you set the subnet, add all of the networks that you want to attach to your machines to the `Networks` list in `providerSpec`. You must also add the network that the primary subnet is part of to this list. 
```

Not 100% about what you mean by `machines subnet` here.

Reply:
I will update with a link to the definition. Chances are someone else will also be confused by that

Reply:
Ah! `machines subnet` vs. `machinesSubnet`.  👍